src/ServiceManagerBase.sol

```diff
@@ -69,4 +76,40 @@ contract ServiceManagerBase is IServiceManager, OwnableUpgradeable {
     function deregisterOperatorFromAVS(address operator) public virtual onlyRegistryCoordinator {
         delegationManager.deregisterOperatorFromAVS(operator);
     }
+
+    /**
+     * @notice Returns the list of strategies that the operator has potentially restaked on the AVS
+     * @param operator The address of the operator to get restaked strategies for
+     * @dev This function is intended to be called off-chain
+     * @dev No guarantee is made on whether the operator has shares for a strategy in a quorum or uniqueness 
+     *      of each element in the returned array. The off-chain service should do that validation separately
+     */
+    function getOperatorRestakedStrategies(address operator) external view returns (address[] memory) {
```

Comment:
we need to set a gas limit which we will call this with to make sure we're clear about how errors are handled